packages/fxa-settings/src/components/ButtonBack/index.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { ReactComponent as BackArrow } from './back-arrow.svg';
+import { useFtlMsgResolver } from '../../models';
+
+interface ButtonBackProps {
+  onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  dataTestId?: string;
+  localizedLabel?: string;
+}
+
+// This component assumes a parent container with class 'relative flex'
+// and an adjacent text element.
```

Comment:
Note that this could be taken one step further and refactored to include the container/adjacent text for both pages that use this component; the reason I didn't here is because I noticed this back button being present last minute and saw the parent and adjacent text have different classes, and both of those both would be additional props. I can rework it here if desired though.